package.json

```
@@ -7,6 +7,7 @@
     "start": "node index.js",
     "dev": "nodemon index.js",
     "test": "NODE_OPTIONS=--experimental-vm-modules jest",
+    "format": "npx prettier . --write --cache",
```

Nit: `npx` not required since it'll be run via the npm script.

Non-nit: Not keen on using `.` as that'll format and save *the entire project*, and I'm very sure most people would only want to format one file at a time (if they choose to do so via the CLI). Instead, we can just have the below and have people pass in the file path individually e.g. `npm run format -- new-era-commands/slash/help.js`

```suggestion
    "format": "prettier --cache --write",
```

---

Oops, yeah copy pasted this from the one I usually use for personal projectss